articles/cognitive-services/cognitive-services-development-options.md

```diff
@@ -49,14 +49,14 @@ Cognitive Services client libraries and REST APIs provide you direct access to y
 * **UI**: N/A - Code only
 * **Subscription(s)**: Azure account + Cognitive Services resources
 
-If you want to learn more about available client libraries and REST APIs, use our [Cognitive Services overview](index.yml) to pick and service and get started with one of our quickstarts for vision, decision, language, and speech.
+If you want to learn more about available client libraries and REST APIs, use our [Cognitive Services overview](index.yml) to pick a service and get started with one of our quickstarts for vision, decision, language, and speech.
 
 ### Cognitive Services for Big Data
 
-With Cognitive Services for Big Data you can embed continuously improving, intelligent models directly into Apache Spark&trade; and SQL computations. These tools liberate developers from low-level networking details, so that they can focus on creating smart, distributed applications. Cognitive Services for Big Data supports the following platforms and connectors: Azure Databricks, Azure Synapse, Azure Kubernetes Service, and Data Connectors.
+With Cognitive Services for Big Data you can embed continuously improving, intelligent models directly into Apache Spark&trade; and SQL computations. These tools liberate developers from low-level networking details, so that they can focus on creating smart, distributed applications. Cognitive Services for Big Data support the following platforms and connectors: Azure Databricks, Azure Synapse, Azure Kubernetes Service, and Data Connectors.
 
 * **Target user(s)**: Data scientists and data engineers
-* **Benefits**: The Azure Cognitive Services for Big Data lets users channel terabytes of data through Cognitive Services using Apache Spark&trade;. It's easy to create large-scale intelligent applications with any datastore.
+* **Benefits**: The Azure Cognitive Services for Big Data let users channel terabytes of data through Cognitive Services using Apache Spark&trade;. It's easy to create large-scale intelligent applications with any datastore.
```

Comment:
I know that we treat Cognitive Services as one thing (internal), but grammatically its plural and now this reads weird.